MapboxNavigation/Resources/nl.lproj/Abbreviations.csv.plist

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>abbreviations</key>
+    <string>&quot;meer&quot;,&quot;Mr&quot;
+&quot;centrum&quot;,&quot;Cntrm&quot;
+&quot;nationaal&quot;,&quot;Nat’l&quot;
+&quot;toetreden&quot;,&quot;Ttrdn&quot;
+&quot;berg&quot;,&quot;Brg&quot;
+&quot;kruising&quot;,&quot;Krsng&quot;
+&quot;downtown&quot;,&apos;&apos;Dtwn&quot;
+&quot;internationaal&quot;,&quot;Int’l&quot;
+&quot;park&quot;,&quot;Pk&quot;
+&quot;plein&quot;,&quot;Pln&quot;
+&quot;heiligen&quot;,&quot;Hlgn&quot;
+&quot;hoogtes&quot;,&quot;Hgts&quot;
+&quot;route&quot;,&quot;Rte&quot;
+&quot;heilige&quot;,&quot;Hlg&quot;
+&quot;markt&quot;,&quot;Mrkt&quot;
+&quot;centrum&quot;,&quot;Cntrm&quot;
+&quot;william&quot;, &quot;Wm&quot;
+&quot;school&quot;,&quot;Sch&quot;
+&quot;senior&quot;,&quot;Snr&quot;
+&quot;rivier&quot;,&quot;Rvr&quot;
+&quot;zus&quot;,&quot;Zus&quot;
+&quot;drop&quot;,&quot;Drp&quot;
+&quot;station&quot;,&quot;Sttn&quot;
+&quot;appartementen&quot;,&quot;Apptn&quot;
+&quot;universiteit&quot;,&quot;Uni&quot;
+&quot;gehucht&quot;,&quot;Hucht&quot;
+&quot;meer&quot;,&quot;Mr&quot;
+&quot;junior&quot;,&quot;Jnr&quot;
+&quot;vader&quot;,&quot;Vdr&quot;
+&quot;punt&quot;,&quot;Punt&quot;
+&quot;gedenkteken&quot;,&quot;Teken&quot;
+&quot;knooppunt&quot;,&quot;Kpunt&quot;
+</string>
+    <key>classifications</key>
+    <string>&quot;rechtbank&quot;,&quot;Rbank&quot;
+&quot;bypass&quot;,&quot;Pass&quot;
+&quot;rijden&quot;,&quot;Rijd&quot;
+&quot;snelweg&quot;,&quot;Weg&quot;
+&quot;brug&quot;,&quot;Br&quot;
+&quot;plaats&quot;,&quot;Plts&quot;
+&quot;halve maan&quot;,&quot;Maan&quot;
+&quot;parkway&quot;,&quot;Pky&quot;
+&quot;rijbaan&quot;,&quot;Strook&quot;
+&quot;laan&quot;,&quot;Ln&quot;
+&quot;autoweg&quot;,&quot;Weg&quot;
+&quot;voetpad&quot;,&quot;Stoep&quot;
+&quot;snoek&quot;,&quot;Snk&quot;
+&quot;snelweg&quot;,&quot;Weg&quot;
+&quot;straat&quot;,&quot;Str&quot;
```

Comment:
@fabiangiger, I took the liberty of capitalizing the abbreviations for consistency with the street names that these abbreviations would be inserted into. I also removed a few lines in [this message](https://www.transifex.com/mapbox/mapbox-navigation-ios/translate/#nl/LocalizableAbbreviations/114199682), such as `"voetpad","stoep"`, which appeared to be two unrelated words rather than an abbreviation pair.

Note that there’s no need for a one-to-one correspondence between English and Dutch within this message. The Dutch translation should have an entirely Dutch list of abbreviations, which is why it’s a single freeform message rather than being spread out across multiple messages. For reference, here’s the context that’s in Transifex for this message:

> Based on this list of road types, the SDK shortens long road names to fit in the turn banner at the top of the screen. Each line in the list includes a lowercase, spelled-out word followed by its title-cased abbreviation. Please pay close attention to the syntax.
> 
> For best results, only include words commonly found in road names, and only use abbreviations that a user would recognize instantly. Feel free to add your own words and remove words that aren’t abbreviated in your language.